lib/enum_chron.rb

```diff
@@ -18,11 +18,10 @@ def enum_chron=(enum_chron)
 
   def normalize_enum_chron
     # normalize into separate n_enum and n_chron
-    unless enum_chron.nil?
-      ec_parser = EnumChronParser.new
-      ec_parser.parse(enum_chron)
-      self.n_enum  = ec_parser.normalized_enum
-      self.n_chron = ec_parser.normalized_chron
-    end
+    enum_chron = self.enum_chron || ""
```

Comment:
Is `enum_chron` here a local that's shadowing `self.enum_chron`? I'm a little confused about why it's written this way -- `self.enum_chron ||= ""` might be more clear if that's the intention

Reply:
`self.enum_chron ||= ""` within `normalize_enum_chron` will blow up because `def enum_chron=` calls `normalize_enum_chron`. Made irrelevant shortly.